.github/renovate.json5

```diff
@@ -22,7 +22,8 @@
     "config:base",
     // Required to not pin dependencies to _exact_ versions (pip)
     ":preserveSemverRanges",
-    "group:monorepos",
+    // Group all updates in one PR
+    "group:all",
```

Comment:
I tend to agree with @snazy that _not_ grouping helps with fixing breakage related to version bumps.

Reply:
As we don't have consensus on grouping, I just keep label for now.